src/components/purchase-complete/index.module.scss

```diff
@@ -31,6 +31,8 @@
   }
   button {
       margin: 10px 5px 0 5px;
+      background-color: var(--color_input_button_background_accent) !important;
```

Comment:
i do think that css var name could be simplified ?
like --**color_input_button_background_accent** to color_button_background_color?